### DISCUSSION TITLE
How To Add New Admins

### DISCUSSION THREAD

**Discussion by michaelgiesen** [snapshot-labs/snapshot#1109] [Q&A] [answered]
Is there a way to add users as an `admin` to a snapshot space?

**Comment by ChaituVR** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/15967809/143180417-05a984a9-ae87-4e67-9f46-f491a666d9eb.png)
If you are the controller of the space, you should be able to see this field in your space settings
